src/ApiCompat/Microsoft.DotNet.ApiCompatibility/Rules/CannotChangeVisibility.cs

```diff
@@ -24,6 +25,40 @@ public CannotChangeVisibility(RuleSettings settings, IRuleRegistrationContext co
             context.RegisterOnTypeSymbolAction(RunOnTypeSymbol);
         }
 
+        private static Accessibility normalizeInternals(Accessibility a) => a switch
```

Comment:
Please start following the [C# Coding Coventions](https://learn.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions) sooner than later. Even though the method is private, we use `CamelCase` in C#.

Reply:
If you change this, it might also be good to comment these around why this method exists.
Something like: When not comparing internals remove internal from the Accesssibility value.  You might also consider calling it "RemoveInternalAccessibility".